src/components/Chips/Chip.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Remove } from "../../assets";
+import "./chip.css";
+
+export const Chip = ({ label, dispatch, id, chipStyle }) => {
```

Comment:
in notecard file you wrote "lables" and here you are writing label, correct the earlier one

Reply:
I am passing it as prop